app/views/runs/show.slim

```diff
@@ -35,6 +35,7 @@
       - if @run.video_url.present?
         li = link_to 'proof', @run.video_url
       li = link_to 'stats', run_stats_path(@run)
+      li = link_to 'graphs', run_graphs_path(@run)
```

Comment:
this little nav bar is getting potentially a bit crowded now

<img width="599" alt="screenshot 2016-08-09 09 47 24" src="https://cloud.githubusercontent.com/assets/438911/17525145/571c45a4-5e16-11e6-85e1-e54f0a65b7b0.png">

maybe putting graphs on the stats page would make sense, since they're similar and the stats page doesn't use much vertical space. i'm open to other ideas on how to reduce the crowding though, i'm not attached to that one.